opencti-platform/opencti-front/src/private/components/data/stream/StreamCollectionCreation.tsx

```diff
@@ -225,6 +203,7 @@ const StreamCollectionCreation: FunctionComponent<StreamCollectionCreationProps>
                 styleNumber={2}
                 redirection
                 searchContext={{ entityTypes: ['Stix-Core-Object', 'stix-core-relationship'] }}
+                entityTypes={['Stix-Core-Object', 'stix-core-relationship', 'Stix-Filtering']}
```

Comment:
What is the difference between `searchContext `and `entityTypes `for `FilterIconButton` ? Sometimes we have one of them only, sometimes they are the same, sometimes not.

Reply:
searchContext is used to look for the available entities to display in some filter lists that display entities (exemple: creator filter, used in useSearchEntities) / entityTypes is the entity types whose filter keys we want to fetch from the schema